presto-tpcds/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.facebook.presto</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>0.152-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-tpcds</artifactId>
+    <description>Presto - TPC-DS Connector</description>
+    <packaging>presto-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.teradata.tpcds</groupId>
+            <artifactId>tpcds</artifactId>
+            <version>1.0-SNAPSHOT</version>
```

Comment:
I know I shouldn't depend on a snapshot version but because tpcds is not released yet I don't have a choice. We'll update this before opening a PR to facebook.

As for the versioning, those properties are used whenever the version is used in more than one place, which isn't the case with tpcds. I move the dependency version out to the root pom but it remains hardcoded.